docs/docs/how_to/trim_messages.ipynb

```diff
@@ -70,73 +114,123 @@
     "    HumanMessage(\"what do you call a speechless parrot\"),\n",
     "]\n",
     "\n",
+    "\n",
     "trim_messages(\n",
     "    messages,\n",
-    "    max_tokens=45,\n",
+    "    # Keep the last <= n_count tokens of the messages.\n",
     "    strategy=\"last\",\n",
+    "    # highlight-start\n",
```

Comment:
very nit but I was a little confused as to why these particular parameters are highlighted. another option is to just set
```python
token_counter = ChatOpenAI(model="gpt-4o")  # can also pass a custom token counter
max_tokens=45  # adjust to desired history length
```
right above `trim_messages(...)`

Reply:
Got it